### PR TITLE
Cs/#889 pooling

### LINKED ISSUE #889: test shared ride hail
add new RH Manager that arbitrarily pools people together. do this simply to confirm that the mechanics of the trips work once they are pooled.

### PR DESCRIPTION
**Pull request by colinsheppard**
Fixed #889

This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/964)

### CODE REVIEW

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

File: src/test/scala/beam/integration/ridehail/allocation/examples/DummyRideHailDispatchWithBufferingRequests.scala [hunk 1]
```diff
@@ -1,74 +1,74 @@
...
+//
+//            rideHailManager.removeDummyRequest(request)
+//
+//            bufferedRideHailRequests.registerVehicleAsReplacementVehicle(rhl.vehicleId)
+//          case None =>
+//        }
+//      }
+//    }
+//  }
+//}
```
Comment (michaz) [inline src/test/scala/beam/integration/ridehail/allocation/examples/DummyRideHailDispatchWithBufferingRequests.scala]: What's the status of this code?

**Pull request merged**